sdk/storage/azure-storage-file-share/azure/storage/fileshare/aio/_directory_client_async.py

```diff
@@ -247,7 +247,7 @@ def get_file_client(self, file_name: str, **kwargs: Any) -> ShareFileClient:
 
         _pipeline = AsyncPipeline(
             transport=AsyncTransportWrapper(self._pipeline._transport),  # pylint: disable=protected-access
-            policies=self._pipeline._impl_policies  # pylint: disable=protected-access
+            policies=self._pipeline._impl_policies  # type: ignore # pylint: disable=protected-access
```

Comment:
Same comment as below: Is `# type: ignore` sufficient, or should I do something more specific like `# type: ignore [arg-type]`